src/ucp/core/ucp_mm.c

```diff
@@ -642,3 +643,47 @@ void ucp_frag_mpool_free(ucs_mpool_t *mp, void *chunk)
 
     ucp_mpool_free(worker, mp, chunk);
 }
+
+void ucp_mem_print_info(ucp_mem_h memh, ucp_context_h context, FILE *stream)
+{
+    unsigned md_index;
+    char buf[256];
```

Comment:
Pls rename buf to smth meaningful or comment what kind of string can be stored here or add a constant with corresponding name.
Actually 32 is enough - 22 for digits + 1 for character + '\0' + alignment ;)